src/main/webapp/resources/css/style.css

```diff
@@ -213,9 +213,6 @@ main{
     opacity: 0.7;
 }
 
-.user-sec{
-    color: #fff;
-}

```

Comment:
Removing this makes the user name in the menu black.

Reply:
I think that maybe a victim of when I was merging your stuff into this branch. Not intended to be removed. Will reinstate.